src/server/routers/account.ts

```diff
@@ -10,7 +10,34 @@ export const accountRouter = t.router({
         },
       },
     });
+    return accounts
+  })
+})
 
+export const isGithubRouter = t.router({
+  me: t.procedure.use(isAuthed).query(async ({ ctx }) => {
```

Comment:
i think you can create one function , and add provider as a parameter  , instead of creating two seperate functions that do the same thing

```suggestion
  me: t.procedure.use(isAuthed)
  .input(
  z.object({provider:z.enum(['github','gitlab'])})
  )
  .query(async ({ ctx }) => {
```

Reply:
i will try to do it